src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -274,6 +278,8 @@ void IIssueConsumer.Accept(string path, IEnumerable<IAnalysisIssue> issues)
 
             var filteredIssues = RemoveSuppressedIssues(issues);
 
+            // See bug #1487: we should be creating the issue markers using the text snapshot
```

Comment:
This is a newly-created bug for a pre-existing issue. I've added it to the current sprint since (1) we're going to have to change this code anyway and (2) the new UX will increase the interval during which the bug could appear since the background analysis will run for longer.